tests/assets/deployment.template_3.json

```diff
@@ -37,7 +57,70 @@
             "restartPolicy": "always",
             "settings": {
               "image": "mcr.microsoft.com/azureiotedge-simulated-temperature-sensor:1.0",
-              "createOptions": "{}"
+              "createOptions": {
```

Comment:
We could still keep some test cases with string value. We should support both types in template